tools/populate_examiners_psc_rel.php

```diff
@@ -0,0 +1,213 @@
+    // Compare without spaces, without cases
+    $examinersOrganized[strtolower(str_replace(' ','',$row['full_name']))][] = $item;
+}
+
+print_r($examinersOrganized);
+
+echo "\nA copy of the `examiners` table has been created as `examiners_17.0`. \n\n";
+
+
+foreach ($examinersOrganized as $name=>$data_array) {
+    echo "Changes for Examiner: ".$name."\n";
+    // the first value will be maintained as the final examiner entry
+    echo "    -> Site ".$data_array[0]['centerID']." added\n";
+    $db->insert('examiners_psc_rel', $data_array[0]);
+    $finalExaminerID =$data_array[0]['examinerID'];
+    unset($data_array[0]);
+
+
+    // all other entries for this examiner will be deleted
+    // certification table and instruments will be updated
+    foreach ($data_array as $k=>$dupData) {
+        // replace old EID by final one
+        $oldExaminerID         =$dupData['examinerID'];
+        $dupData['examinerID'] =$finalExaminerID;
+
+        //INSERT into examiners_psc_rel
+        echo "    -> Site ".$dupData['centerID']." added\n";
+        $db->insertIgnore('examiners_psc_rel', $dupData);
+
+        // UPDATE certification table
+        echo "        -> Updating Certification examinerIDs: ".$oldExaminerID."=>".$finalExaminerID."\n";
+        $db->update('certification', array('examinerID' => $finalExaminerID), array('examinerID' => $oldExaminerID));
+
+        // UPDATE final_radiological_review
+        echo "        -> Updating final radiological review examinerIDs: ".$oldExaminerID."=>".$finalExaminerID."\n";
+        $db->update('final_radiological_review', array('Final_Examiner' => $finalExaminerID), array('Final_Examiner' => $oldExaminerID));
+        $db->update('final_radiological_review', array('Final_Examiner2' => $finalExaminerID), array('Final_Examiner2' => $oldExaminerID));
+        // UPDATE instrument tables
+        foreach ($instruments as $table=>$name) {
+            if ($db->tableExists($table) && $db->columnExists($table, 'Examiner')) {
+                echo "        -> Updating Instrument ".$table." examinerIDs: ".$oldExaminerID."=>".$finalExaminerID."\n";
+                $db->update($table, array('Examiner' => $finalExaminerID), array('Examiner' => $oldExaminerID));
+            }
+        }
+        // DELETE examiners additional entries
+        echo "    -> DELETING duplicate examinerID: ".$oldExaminerID."\n\n";
+        $db->delete('examiners', array('examinerID' => $oldExaminerID));
+    }
+}
+
+echo "\n\nThe script ran successfully, make sure to run the following clean-up commands.\n
+These commands will drop the now redundant fields and add new constraints to the examiner tables.\n\n
+-- ALTER TABLE examiners DROP FOREIGN KEY FK_examiners_1;\n
+-- ALTER TABLE examiners DROP INDEX full_name;\n
+-- ALTER TABLE examiners DROP KEY FK_examiners_1;\n
+-- ALTER TABLE examiners DROP COLUMN `CenterID`;\n
+-- ALTER TABLE examiners DROP COLUMN `active`;\n
+-- ALTER TABLE examiners DROP COLUMN `pending_approval`;\n
```

Comment:
If these are just being printed and not executed, why do they start with an SQL comment delimiter? (`--`)? It seems confusing, and people will probably copy/paste it and not realize it doesn't do anything..